internal/fsutil/fsutil.go

```diff
@@ -0,0 +1,40 @@
+package fsutil
```

Comment:
We have now `fsutil.Move` because we may be moving from a fs-watcher or directly from an activity when we're dealing with batches.